Finish implementation of optional attributes (#1537)
yewstack/yew: docs/concepts/html/elements.md

```
<!--END_DOCUSAURUS_CODE_TABS-->

## Optional attributes for HTML elements

Most HTML attributes can be marked as optional by placing a `?` in front of
the `=` sign. This makes them accept the same type of value as otherwise, but
wrapped in an `Option<T>`:

```rust
let maybe_id = Some("foobar");

html! {
    <div id?=maybe_id></div>
}
```

> If the attribute is set to `None`, it will behave as though it wasn't set.

Reviewer: ```suggestion
If the attribute is set to `None`, it attribute won't be set in the DOM.
```

Reviewer: It's *very* minor.

Author: That line stood out to me as well, thanks.
